packages/mobile/src/components/details-tile/DetailsTilePremiumAccess.tsx

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react'
```

Comment:
Not seeing where this component is used in `DetailsTile.tsx`, assuming it's in a later change.